energysmartbridge/src/Modules/MQTTModule.cs

```diff
@@ -143,22 +132,16 @@ protected virtual void OnAppMessage(MqttApplicationMessageReceivedEventArgs e)
                 }
                 else if (topic == Topic.mode_command)
                 {
-                    log.Debug($"Queued {id} Mode: {payload}");
-
-                    string heater_mode = "off";
-                    switch(payload) {
-                        case "heat_pump":
-                            heater_mode = "Efficiency";
-                            break;
-                        case "eco":
-                            heater_mode = "Hybrid";
-                            break;
-                        case "electric":
-                            heater_mode = "Electric";
-                            break;
-                        default:
-                            break;
-                    }
+                    string heater_mode = payload switch
```

Comment:
Since you seem way more familiar with C#, is there a way we can reuse this mapping both ways without having to repeat it here and in the other file?